### PR TITLE
parser: add support for `show pump/drainer status`

### PR DESCRIPTION
**Pull request by caohe**
### What problem does this PR solve? 
Add support for `show pump/drainer status`
[#9201](https://github.com/pingcap/tidb/issues/9201)

### What is changed and how it works?
Add pump and drainer to parser.y and dml.go

### Check List 

Tests 

 - Unit test

Code changes

 - Has interface methods change

Side effects

 - None

Related changes

 - None

### CODE REVIEW

**Comment by caohe** [comment]
@WangXiangUSTC PTAL

**Comment by WangXiangUSTC** [comment]
thanks @caohe

File: go.mod1 [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/pingcap/errors v0.11.0
 	github.com/pingcap/tidb v0.0.0-20190218065808-69472bd1a6e9
 	github.com/pingcap/tipb v0.0.0-20190107072121-abbec73437b7
+	github.com/pkg/errors v0.8.1
```
Comment (WangXiangUSTC) [inline go.mod1]: seems don't need this

**Comment by WangXiangUSTC** [comment]
@kennytm PTAL

File: parser.y [hunk 2]
```diff
@@ -477,8 +477,10 @@ import (
 	buckets		"BUCKETS"
 	cancel		"CANCEL"
 	ddl		"DDL"
+	drainer     "DRAINER"
 	jobs		"JOBS"
 	job		    "JOB"
+	pump        "PUMP"
```
Comment (WangXiangUSTC) [inline parser.y]: please format this line

File: go.mod1 (hunk 1 above)
Comment (caohe) [reply go.mod1]: OK. I have removed it.

File: parser.y (hunk 2 above)
Comment (caohe) [reply parser.y]: Done. PTAL

**Pull request merged**